.swiftlint.yml

```diff
@@ -1,3 +1,43 @@
+custom_rules:
+  missing_swifterswift_prefix:
+    included: ".*\\.swift"
+    excluded: ".*Tests\\.swift"
+    name: "Missing SwifterSwift: prefix"
+    regex: "///\\s(?!SwifterSwift:\\s)\\w+[^\\n]*$"
+    match_kinds:
+      - doccomment
+    message: "Doc string should always start with the SwifterSwift: prefix"
+    severity: warning
+  unnecessary_type:
+    name: "Unnecessary Type"
+    regex: '(?sm)[ \ta-zA-Z0-9]?(?:let|var){1} [ \ta-zA-Z0-9]+?:[ \t]+?([a-zA-Z0-9]+?)[\t ]+?=[\t ]?\1'
```

Comment:
This would not allow things like 
```swift
let s: String = getString()
```
?
That might give false positive because I think sometimes this use of types is necessary to disambiguate expressions.

Reply:
Sure, I'm not saying this rule is perfect 😁, at least it helps in some scenarios like
```swift
let date: Date = Date()
```
In the cases where having the type is required to disambiguate expressions, we can always add an exception, or if we see this rule bothering us a lot in the future, we can remove it of course

Reply:
Sure @omaralbeik :)) I just mentioned it because I see this being used a lot (not here in this project but overall) to disambiguate, specially literals like `let s: NString = ""`  since if we do like `let s = ""` would be infered as String. 
 Of course we can always use `let s = "" as NSString` :))